model-providers/ollama/runtime/src/main/java/io/quarkiverse/langchain4j/ollama/QuarkusOllamaClient.java

```diff
@@ -0,0 +1,46 @@
+package io.quarkiverse.langchain4j.ollama;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
+import org.jboss.resteasy.reactive.client.api.LoggingScope;
+
+import io.quarkus.rest.client.reactive.QuarkusRestClientBuilder;
+import io.smallrye.mutiny.Multi;
+
+public class QuarkusOllamaClient implements OllamaClient {
```

Comment:
Couldn't this just be a rename instead of having deleted `OllamaClient` and add this new one? 
Asking because it would be nice to preserve the Git history

Reply:
OllamaClient is now the interface. As this code was part of langchain it define the SPi with 2 implementations one for langchain and one for quarkus, it will facilitate the move back to langchain4j. The OllamaClient history is still in QuarlusOllamaClient

Reply:
> The OllamaClient history is still in QuarlusOllamaClient

That's my point, it's not because this is considered by Git to be a new class

Reply:
Don't get your point, I've renamed old OllamaClient to QuarkusOllamaClient.
![image](https://github.com/user-attachments/assets/35932da8-ba63-4e67-9561-9e660c3e95fc)